tests/acceptance/features/bootstrap/NotificationContext.php

```diff
@@ -134,4 +153,96 @@ public function theJsonDataFromLastResponseShouldMatch(
 			$this->featureContext->getJSONSchema($schemaString)
 		);
 	}
+
+	/**
+	 * @Then /^the email address of user "([^"]*)" should have received an email for space "([^"]*)" from user "([^"]*)" with the body containing$/
+	 *
+	 * @param string $user
+	 * @param string $spaceName
+	 * @param string $sender
+	 * @param PyStringNode $content
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function theEmailAddressOfUserShouldHaveReceivedAnEmailForSpaceFromUserWithTheBodyContaining(string $user, string $spaceName, string $sender, PyStringNode $content):void {
+		$expectedContent = \str_replace("\r\n", "\n", $content->getRaw());
+		$this->spacesContext->theUserListsAllHisAvailableSpacesUsingTheGraphApi($sender);
```

Comment:
I would suggest using the helper functions to list the spaces not the step defs
```suggestion
	SpacesHelper::listAllTheSpace($sender);
```